intelmq/bots/experts/gethostbyname/expert.py

```diff
@@ -21,29 +21,35 @@
 import socket
 
 from intelmq.lib.bot import Bot
+from intelmq.lib.harmonization import URL
 
 
 class GethostbynameExpertBot(Bot):
 
+    def init(self):
+        self.fallback_to_url = getattr(self.parameters, 'fallback_to_url', True)
```

Comment:
Hm, for backwards compatibility, I'd favor `False` here. I'm fine with the default value of True in BOTS.

Reply:
Clever! Changed and added a comment so that future reviewer gets a chance to understand why we are no using the default value from BOTS.